codegen/lib/graphql_java_gen.rb

```diff
@@ -274,4 +274,24 @@ def abstract_types
       end
     end
   end
+
+  def java_doc(element)
+    doc = ''
+    unless element.description.nil? || element.description == 0
```

Comment:
I don't know of a case when the description could be equal to a literal `0`?  I would think the nil check is all you need.